boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi

```diff
@@ -29,6 +29,28 @@
 		compatible = "nordic,npm1100";
 		nordic,iset-gpios = <&gpio0 23 GPIO_ACTIVE_HIGH>;
 	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		red_pwm_led: pwm_led_0 {
+			pwms = <&pwm0 0 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
+		};
+		green_pwm_led: pwm_led_1 {
+			pwms = <&pwm0 1 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
+		};
+		blue_pwm_led: pwm_led_2 {
+			pwms = <&pwm0 2 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
+		};
+	};
+
+	aliases {
+		pwm-led0 = &red_pwm_led;
```

Comment:
Thanks for the PR! This looks good, but the naming should be changed to correspond to the silk-screen.
We may add PWM to more LEDs and then `<color>_pwm_led` becomes ambiguous.

All LEDs in this PR should refer to "RGB1"

Reply:
I understand, but maybe it makes sense to have the first/primary one keep the names to keep it compatible with e.g. the rgb_led sample (see below).  Should this then be done as an alias for the default RGB LED? I am not a devicetree expert, so please help me out :)

```
static const struct pwm_dt_spec red_pwm_led =
	PWM_DT_SPEC_GET(DT_ALIAS(red_pwm_led));
static const struct pwm_dt_spec green_pwm_led =
	PWM_DT_SPEC_GET(DT_ALIAS(green_pwm_led));
static const struct pwm_dt_spec blue_pwm_led =
	PWM_DT_SPEC_GET(DT_ALIAS(blue_pwm_led));
```

Reply:
You can refer to RGB1 in the label for the node, e.g. define it as `rgb1_red_pwm_led: pwm_led_0 {` and define the standard alias to refer to that label: `red-pwm-led = &rgb1_red_pwm_led;`.